pymc3/distributions/continuous.py

```diff
@@ -832,10 +817,7 @@ class HalfNormal(PositiveContinuous):
             x = pm.HalfNormal('x', tau=1/15)
     """
 
-    def __init__(self, sigma=None, tau=None, sd=None, *args, **kwargs):
-        if sd is not None:
-            sigma = sd
```

Comment:
Instead of deleting, could you raise an error here and advice using sigma

The reason being that `sd` was a keyword for a REALLY long time so there are a lot of examples out there in the wild with this keyword.